src/appsignal/cli.py

```diff
@@ -135,3 +150,54 @@ def _write_file(self):
             )
 
             f.write(file_contents)
+
+
+class DemoCommand(AppsignalCLICommand):
+    def __init__(self, push_api_key=None, application=None):
+        self._push_api_key = push_api_key or os.environ.get("APPSIGNAL_PUSH_API_KEY")
+        self._name = application
+
+    def run(self) -> int:
+        print("Sending example data to AppSignal...")
+
+        if self._push_api_key is None:
+            self._input_push_api_key()
+
+        if self._name is None:
+            self._input_name()
+
+        print()
+
+        client = Client(
+            active=True,
+            name=self._name,
+            push_api_key=self._push_api_key,
+            log_level="trace",
+        )
+        print(f"Starting AppSignal client for {self._name}...")
+        client.start()
+
+        tracer = trace.get_tracer(__name__)
+
+        # Performance sample
```

Comment:
Please add the `demo_sample: true` tag to both samples so the UI will show a special box explaining it's a demo sample. See also [how we do it in Node.js](https://github.com/appsignal/appsignal-nodejs/blob/484e3a161211a2d7d789a52b1efa81df2719d8a9/src/demo.ts#L26).